pallet/ecdsa-authority/src/lib.rs

```diff
@@ -224,15 +212,6 @@ pub mod pallet {
 				Default::default()
 			}
 		}
-
-		fn on_runtime_upgrade() -> Weight {
```

Comment:
It's safe to remove for Crab.

Because we haven't setup any ecdsa authority in Crab yet.

No data on Crab.

Reply:
The current Pangoro on chain runtime version is `6,009`. Does it need the ecdsa migration? Maybe you should upgrade that first.